repository-data/webfiles/src/main/resources/site/src/scss/rebrand/ndrs-frontend-edge-cases.scss

```diff
@@ -0,0 +1,3564 @@
+  color: #353535;
+  font-weight: 600;
+}
+
+.nhsd-t-form-error,
+.nhsd-t-form-label-s {
+  margin: 0 0 1.1111111111rem;
+}
+
+.nhsd-t-quote {
+  position: relative;
+  margin: 0 auto 0.8333333333rem;
+  color: #3f525f;
+  font-size: 1.445rem;
+  letter-spacing: -0.01666rem;
+  line-height: 1.19252;
+}
+
+.nhsd-t-flex--debug {
+  min-height: 100px;
+  background: #d5dade;
+}
+
+.nhsd-t-flex--debug div {
+  padding: 0.5rem 1rem;
+  color: #fff;
+  background: #3f525f;
+  box-shadow: inset 0 0 0 0.5px #353535;
+}
+.nhsd-t-flex-xs--debug div {
+  padding: 0.5rem 1rem;
+  color: #fff;
+  background: #3f525f;
+  box-shadow: inset 0 0 0 0.5px #353535;
+}
+
+.nhsd-t-flex-s--debug {
+  min-height: 100px;
+  background: #d5dade;
+}
+
+.nhsd-t-flex-s--debug div {
+  padding: 0.5rem 1rem;
+  color: #fff;
+  background: #3f525f;
+  box-shadow: inset 0 0 0 0.5px #353535;
+}
+
+.nhsd-t-flex-m--debug {
+  min-height: 100px;
+  background: #d5dade;
+}
+
+.nhsd-t-flex-m--debug div {
+  padding: 0.5rem 1rem;
+  color: #fff;
+  background: #3f525f;
+  box-shadow: inset 0 0 0 0.5px #353535;
+}
+
+.nhsd-a-box {
+  display: block;
+  position: relative;
+  padding: 1.222rem;
+  border-radius: 0.333rem;
+  color: #353535;
+  background: #fff;
+  transition-property: box-shadow;
+  transition-duration: 0.2s;
+}
+
+.nhsd-a-box--bg-black {
+  background: #353535;
+}
+
+.nhsd-a-box--bg-dark-grey {
+  background: #3f525f;
+}
+
+.nhsd-a-box--bg-mid-grey {
+  background: #768692;
+}
+
+.nhsd-a-box--bg-light-grey {
+  background: #edf1f1;
+}
+
+.nhsd-a-box--bg-dark-blue {
+  background: #003087;
+}
+
+.nhsd-a-box--bg-blue {
+  background: #004da8;
+}
+
+.nhsd-a-box--bg-light-blue {
+  background: #cce3f5;
+}
+
+.nhsd-a-box--bg-light-blue-10 {
+  background: #e5f0fa;
+}
+
+.nhsd-a-box--bg-yellow {
+  background: #fae100;
+}
+
+.nhsd-a-box--bg-light-yellow {
+  background: #fef9cc;
+}
+
+.nhsd-a-box--bg-dark-green {
+  background: #006646;
+}
+
+.nhsd-a-box--bg-red {
+  background: #b30f0f;
+}
+
+.nhsd-a-box--border-yellow {
+  box-shadow: inset 0 0 0 0.115rem #fae100;
+}
+
+.nhsd-a-box--border-grey {
+  box-shadow: inset 0 0 0 0.115rem #d5dade;
+}
+
+.nhsd-a-box--border-blue {
+  box-shadow: inset 0 0 0 0.115rem #41b6e6;
+}
+
+.nhsd-a-box--border-green {
+  box-shadow: inset 0 0 0 0.115rem #009639;
+}
+
+.nhsd-a-box--border-red {
+  box-shadow: inset 0 0 0 0.115rem #b30f0f;
+}
+
+.nhsd-a-icon--col-black {
+  fill: #353535;
+}
+
+.nhsd-a-icon--col-dark-grey {
+  fill: #3f525f;
+}
+
+.nhsd-a-icon--col-mid-grey {
+  fill: #768692;
+}
+
+.nhsd-a-button .nhsd-a-icon > svg,
+.nhsd-a-icon--col-white {
+  fill: #fff;
+}
+
+.nhsd-a-icon--col-yellow {
+  fill: #fae100;
+}
+
+.nhsd-a-icon--col-blue {
+  fill: #1d6162 !important;
+}
+
+.nhsd-a-icon--col-red {
+  fill: #da291c;
+}
+
+.nhsd-a-icon--col-green {
+  fill: #009639;
+}
+.nhsd-m-card__content_container {
+  .ndrs-icon-hover {
+    border-radius: 50%;
+    border: 3px solid transparent;
+  }
+  &:hover {
```

Comment:
This does not look like edge case. You should been using the https://design-system.digital.nhs.uk/. Please put `ndrs-frontend-edge-cases.scss` back to what is was. You will have to move your CSS to an NDRS folder and give your CSS file a better name.

Also, you are loading `nhsuk.css`  on your NDRS pages.  This should be removed, as the old branding is soon to be delete.